cmd/notary-server/config.go

```diff
@@ -178,15 +178,18 @@ func getTrustService(configuration *viper.Viper, sFactory signerFactory,
 		return nil, "", err
 	}
 
-	minute := 1 * time.Minute
+	duration := 10 * time.Second
```

Comment:
I'm not sure if 10 seconds is a proper duration, just seems better than one minute.